runtime/model-protobuf/src/test/java/io/aklivity/zilla/runtime/model/protobuf/internal/ProtobufParserTest.java

```diff
@@ -14,254 +14,318 @@
  */
 package io.aklivity.zilla.runtime.model.protobuf.internal;
 
-import static org.junit.Assert.*;
-
+import static io.aklivity.zilla.runtime.model.protobuf.internal.ProtobufParser.ProtoSyntax.PROTO2;
+import static io.aklivity.zilla.runtime.model.protobuf.internal.ProtobufParser.ProtoSyntax.PROTO3;
+import static io.aklivity.zilla.runtime.model.protobuf.internal.ProtobufParser.ProtoSyntax.UNKNOWN;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
 import org.junit.Test;
 
-import com.google.protobuf.Descriptors.FileDescriptor;
 import com.google.protobuf.Descriptors.Descriptor;
+import com.google.protobuf.Descriptors.EnumDescriptor;
 import com.google.protobuf.Descriptors.FieldDescriptor;
-
+import com.google.protobuf.Descriptors.FileDescriptor;
 
 public class ProtobufParserTest
 {
+    private ProtobufParser parser;
+
+    @Before
+    public void setUp()
+    {
+        parser = new ProtobufParser(null);
+    }
+
+
     @Test
-    public void shouldParseProto2Schema()
+    public void shouldDetectProto2Syntax()
     {
-        // Given: A proto2 schema
-        String proto2Schema =
-                "syntax = \"proto2\";\n" +
-                        "package example;\n" +
-                        "\n" +
-                        "message Person {\n" +
-                        "  required string name = 1;\n" +
-                        "  optional int32 age = 2;\n" +
-                        "  repeated string email = 3;\n" +
-                        "}\n";
-
-        // When: Parsing with the unified parser
-        ProtobufParser parser = new ProtobufParser(new FileDescriptor[0]);
-        FileDescriptor descriptor = parser.parse(proto2Schema);
-
-        // Then: The schema should be correctly parsed
-        assertNotNull(descriptor);
-        assertEquals("proto2", descriptor.getSyntax());
-        assertEquals("example", descriptor.getPackage());
-        assertEquals(1, descriptor.getMessageTypes().size());
+        String schema = """
+                syntax = "proto2";
+                message Test {}""";
```

Comment:
The closing `"""` can go on the next line too, so that the end of the string is more readable without the `"""` right next to it.

Feedback applies to all multiline strings.